App/Persistence/TemporaryDbProvider.cs

```diff
@@ -14,9 +14,15 @@ public Canvas GetCanvas()
     {
         return _canvas;
     }
+    public void AddPixel(Pixel pixel)
+    {
+        _canvas.Pixels.Append(pixel);
```

Comment:
Use the method Add to connect the new object to the list. The Append method will "show the list with the added member" but not save it.
```suggestion
        _canvas.Pixels.Add(pixel);
```